README.md

```diff
@@ -507,7 +512,7 @@ const records = await arIO.getArNSRecords();
   },
   "ar-io": {
     "contractTxId": "eNey-H9RB9uCdoJUvPULb35qhZVXZcEXv8xds4aHhkQ",
-    "purchasePrice": 17386.717520731843,
+    "purchasePrice": 17386.717520731843, // value in mIO
```

Comment:
Should maybe update this readme example as I think this wouldn't be a typical amount of mIO for purchasePrice